lib/resources/app_theme.dart

```diff
@@ -20,6 +20,11 @@ class AppTheme {
             fontSize: 15,
             fontWeight: FontWeight.w400,
           ),
+          subtitle1: TextStyle(
+            color: Colors.white,
+            fontSize: 26,
+            fontWeight: FontWeight.w800,
```

Comment:
you may want to check the `typography.dart` file for the default fontSize and fontWeight to manage your own theme with more sense. For example, bodyText1 is usually at the same size as the subtitle1. So, given that a `bodyText1` at size 16, then you may better consider size 26 == headline5.

Reply:
Fixed https://github.com/luongvo/flutter-survey/pull/49/commits/1abe0b2bcc9e5d9a5143c15dda51929467fdc212